edge-util/src/Microsoft.Azure.Devices.Edge.Storage/EncryptedStore.cs

```diff
@@ -9,14 +9,15 @@ namespace Microsoft.Azure.Devices.Edge.Storage
     public class EncryptedStore<TK, TV> : IKeyValueStore<TK, TV>
```

Comment:
This change will work for encrypting twin as well?

Reply:
Yes, this and the change in UpdatableEncryptedStore